gdsfactory/cell.py

```diff
@@ -53,36 +54,42 @@ class Settings(BaseModel):
     child: dict[str, Any] | None = None
 
 
-def cell(func: _F) -> _F:
-    """Decorator for Component functions.
-
-    Implements a cache so that if a component has already been build
-    it will return the component from the cache directly.
+def cell(
+    func: _F | None = None,
+    /,
+    *,
+    autoname: bool = True,
+    max_name_length: int = CONF.max_name_length,
+    include_module: bool = False,
+    with_hash: bool = False,
+    ports_off_grid: str = CONF.ports_off_grid,
+    ports_not_manhattan: str = CONF.ports_not_manhattan,
+    flatten: bool = False,
+    naming_style: str = "default",
+    default_decorator: Callable[[Component], Component] | None = None,
+) -> Callable[[_F], _F]:
+    """Parametrized Decorator for Component functions.
+
+    decorator_kwargs are the default settings that the decorator will use, they can be overridden by the decorated function's kwargs.
```

Comment:
`decorator_kwargs` doesn't appear anywhere else in this file - is it defined elsewhere? Or is it referring to the keyword arguments on this method?

If it's the latter, I recommend spelling it out (e.g. "Keyword args on this decorator..."). As it stands, `decorator_kwargs` looks like a code object.

Reply:
good catch, i removed it

https://github.com/gdsfactory/gdsfactory/pull/2219/commits/9bfe56f6b3e1753a690b149116fbd08717c931f0